data-raw/amazon_books.R

```diff
@@ -0,0 +1,8 @@
+## code to prepare `amazon-books` dataset goes here
```

Comment:
This entire code should go in `data-raw/process_data_sets.R`, not a new script. Please look there for examples.